native/src/routes/Settings.tsx

```diff
@@ -21,7 +21,7 @@ type SettingsProps = {
 const Settings = ({ navigation }: SettingsProps): ReactElement => {
   const appContext = useCityAppContext()
   const showSnackbar = useSnackbar()
-  const { t } = useTranslation('settings')
+  const { t } = useTranslation<['settings', 'layout']>(['settings', 'layout'])
```

Comment:
🙃 Is the type annotation actually necessary?
```suggestion
  const { t } = useTranslation(['settings', 'layout'])
```
Edit: If you do `t('layout:my-key')` anyway, the previous `useTranslation('settings')` would be enough. You only need to pass the additional namespace if you want to use `layout:my-key` like this: `t('my-key')`. I think in this case the better choice would be to do it explicitly, so you can just revert this here:
```suggestion
  const { t } = useTranslation('settings')
```